.github/workflows/release.yml

```diff
@@ -56,12 +56,13 @@ jobs:
           echo "release_version=${release_version}" >> "$GITHUB_ENV"
           export ORG_GRADLE_PROJECT_releaseVersion="${release_version}"
           make buildplugin
-          sha256sum ./protoc-gen-connect-kotlin/build/libs/protoc-gen-connect-kotlin-${release_version}.jar >> sha256.txt
+          mv ./protoc-gen-connect-kotlin/build/libs/protoc-gen-connect-kotlin-${release_version}.jar .
+          sha256sum protoc-gen-connect-kotlin-${release_version}.jar >> sha256.txt
```

Comment:
This will fix errors like this (seen with previous v0.1.4 release):

```
$ sha256sum -c sha256.txt
sha256sum: ./protoc-gen-connect-kotlin/build/libs/protoc-gen-connect-kotlin.jar: No such file or directory
./protoc-gen-connect-kotlin/build/libs/protoc-gen-connect-kotlin.jar: FAILED open or read
sha256sum: WARNING: 1 listed file could not be read
```